swig/openscap_api.py

```diff
@@ -31,31 +31,23 @@
 
 logger = logging.getLogger("openscap")
 
-from sys import version_info
-if version_info >= (2, 6, 0):
-    def _import_helper():
-        from os.path import dirname
-        import imp
-        fp = None
-        try:
-            fp, pathname, description = imp.find_module(
-                '_openscap_py', [dirname(__file__)])
-        except ImportError:
-            import _openscap_py as OSCAP
-            return OSCAP
-        if fp is not None:
-            try:
-                _mod = imp.load_module(
-                    '_openscap_py', fp, pathname, description)
-            finally:
-                fp.close()
-            return _mod
-    OSCAP = _import_helper()
-    del _import_helper
-else:
-    import _openscap_py as OSCAP
-
-del version_info
+
+def _import_helper():
+    from os.path import dirname
+    import importlib.machinery
+    import importlib.util
+    spec = importlib.machinery.PathFinder.find_spec('_openscap_py', [dirname(__file__)])
+    if not spec:
+        spec = importlib.machinery.PathFinder.find_spec('_openscap_py')
+    if not spec:
+        return None
```

Comment:
Should we raise an exception/print a message that import failed?

Reply:
We'll get one after trying to access module which is None. Traceback is pretty much self-explanatory.

Reply:
But I can change that into `raise ImportError("Unable to import _openscap_py module, extra path: %s." % dirname(__file__))`